File: dayOne/java/levelTwo/MoveZeros.java
public class MoveZeros {

    public static void moveZerosToBackOfArray(int[] array) {

        int nonZeroIndex = 0;

        for (int index = 0; index < array.length; index++) {
            if (array[index] != 0) {
                int temporaryValue = array[index];
                array[index] = array[nonZeroIndex];
                array[nonZeroIndex++] = temporaryValue;
            }
        }
    }
}

File: dayOne/python/levelTwo/move_zeros.py
def move_zeros_in_list(array):
    non_zero_index = 0

    for index in range(len(array)):

        if array[index] != 0:
            temporary_value = array[index]
            array[index] = array[non_zero_index]
            array[non_zero_index] = temporary_value
            non_zero_index += 1